aviary/interface/methods_for_level2.py

```diff
@@ -1701,103 +1701,77 @@ def add_design_variables(self):
             # vehicle sizing problem
             # size the vehicle (via design GTOW) to meet a target range using all fuel capacity
             if self.problem_type is ProblemType.SIZING:
                 self.model.add_design_var(
                     Mission.Design.GROSS_MASS,
-                    lower=10.,
-                    upper=400.e3,
-                    units="lbm",
-                    ref=175_000,
+                    lower=10.0,
+                    upper=400e3,
+                    units='lbm',
+                    ref=175e3,
                 )
                 self.model.add_design_var(
                     Mission.Summary.GROSS_MASS,
-                    lower=10.,
-                    upper=400.e3,
-                    units="lbm",
-                    ref=175_000,
+                    lower=10.0,
+                    upper=400e3,
+                    units='lbm',
+                    ref=175e3,
                 )
 
-                self.model.add_constraint(
-                    Mission.Constraints.RANGE_RESIDUAL, equals=0, ref=10
-                )
                 self.model.add_subsystem(
-                    "gtow_constraint",
+                    'gtow_constraint',
                     om.EQConstraintComp(
-                        "GTOW",
-                        eq_units="lbm",
+                        'GTOW',
+                        eq_units='lbm',
                         normalize=True,
                         add_constraint=True,
                     ),
                     promotes_inputs=[
-                        ("lhs:GTOW", Mission.Design.GROSS_MASS),
-                        ("rhs:GTOW", Mission.Summary.GROSS_MASS),
+                        ('lhs:GTOW', Mission.Design.GROSS_MASS),
+                        ('rhs:GTOW', Mission.Summary.GROSS_MASS),
                     ],
                 )
 
+                self.model.add_constraint(
+                    Mission.Constraints.RANGE_RESIDUAL, equals=0, ref=10
+                )
+
             # target range problem
             # fixed vehicle (design GTOW) but variable actual GTOW for off-design mission range
             elif self.problem_type is ProblemType.ALTERNATE:
                 self.model.add_design_var(
                     Mission.Summary.GROSS_MASS,
                     lower=0,
                     upper=None,
-                    units="lbm",
-                    ref=175_000,
+                    units='lbm',
+                    ref=175e3,
                 )
 
                 self.model.add_constraint(
-                    Mission.Constraints.RANGE_RESIDUAL, equals=0, ref=10,
+                    Mission.Constraints.RANGE_RESIDUAL, equals=0, ref=10
                 )
+
             elif self.problem_type is ProblemType.FALLOUT:
                 print('No design variables for Fallout missions')
 
+            if self.mission_method is TWO_DEGREES_OF_FREEDOM:
+                if self.analysis_scheme is AnalysisScheme.COLLOCATION:
```

Comment:
this could just be 
`if (self.mission_method is TWO_DEGREES_OF_FREEDOM) and (self.analysis_scheme is AnalysisScheme.COLLOCATION):`
right?

Reply:
I have absolutely no idea why, but making the change you suggested causes `AviaryMissionTimeseries.test_timeseries_report` to fail. The test passes if I leave it as is.

Reply:
I think #433 should have fixed this

Reply:
Could you try re-doing this change and pulling in main to see if it passes?